ukvm/ukvm_module_net.c

```diff
@@ -109,13 +107,14 @@ static int tap_attach(const char *ifname)
         errno = ENOENT;
         return -1;
     }
-    if (!up) {
+
+#if defined(__linux__)
+
+    if (! (ifp->ifa_flags & (IFF_UP | IFF_RUNNING))) {
```

Comment:
force-pushed... interesting that no C compiler told me about this use-after-free...